### PR TITLE
man: add ifcfg-lo.5 manual page

### PR DESCRIPTION
**Pull request by MikhailKasimov**
- Added ```ifcfg-lo.5.in``` man-page to be full of ifcfg types descriptions.

Not entirely sure on text padding is correct (text is written manually), feel free to fix it, if needed. Thanks!

### CODE REVIEW

**Comment by mtomaschewski** [comment]
BTW: We are thinking about a rewrite of all the man pages to  asciidoc (for example), which is probably easier to write and read and allows to generate not only normal man page output, but also e.g. html or pdf.

File: man/ifcfg-lo.5.in [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   USERCONTROL=no
+   FIREWALL=no
+
+.fi
+
+\fBNote\fR: Loopback is automatically created and set up by the Linux kernel.
+\fBifcfg\fR file marks it \fBSTARTMODE=nfsroot\fR to skip a shutdown on network \fBifdown\fR requests.
+
+.SH BUGS
+Please report bugs at <http://www.suse.de/feedback>
```
Comment (mtomaschewski) [inline man/ifcfg-lo.5.in]: http://bugs.opensuse.org

Comment (MikhailKasimov) [reply man/ifcfg-lo.5.in]: Ok, let it be so.

**Comment by mtomaschewski** [comment]
Please add a commit adding it to Makefile.am [no, automake does  not support wildcardrs ;-)]
```
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -30,8 +30,9 @@ generated                     = \
        ifcfg-vlan.5            \
        ifcfg-vxlan.5           \
        ifcfg-wireless.5        \
-       ifcfg.5                 \
        ifcfg-dhcp.5            \
+       ifcfg-lo.5              \
+       ifcfg.5                 \
        ifroute.5               \
        ifrule.5                \
        ifsysctl.5              \
```

**Comment by mtomaschewski** [comment]
squashed  and merged manually with additional commit adding it to make and spec file

**Pull request closed without merging**